components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -205,11 +199,11 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 	return false, nil
 }
 
-func (r *reconciler) deleteExternalDependency(ctx context.Context, subscription *eventingv1alpha1.Subscription, channelName string, channelNamespace string) error {
+func (r *reconciler) deleteExternalDependency(ctx context.Context, knativeSubsName string, knativeSubsURI string, channelName string, namespace string) error {
```

Comment:
The parameter "knativeSubsURI string" is not needed, cause if a knSubscriptoin exists, the the knSubscriberURI can be obtained from it:
```
 knSubscriberURI := knativeSubs.Spec.Subscriber.DNSName
```

Reply:
Done